src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,119 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final int MIN_AGE = 18;
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    private static final RegistrationService regService = new RegistrationServiceImpl();
+    private User defaultUser;
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();

```

Comment:
```suggestion
```

Reply:
I want to add each User in tests to empty Storage, that is why I clear Storage before every test

Reply:
I think he was talking about moving it out to the after each method instead, this way we would have it more appropriate, so let's do this